godtools/App/Features/Onboarding/Views/OnboardingTutorialIntroViewModel.swift

```diff
@@ -10,14 +10,42 @@ import UIKit
 
 class OnboardingTutorialIntroViewModel: OnboardingTutorialIntroViewModelType {
 
+    private weak var flowDelegate: FlowDelegate?
+    
+    private let analyticsContainer: AnalyticsContainer
+    private let analyticsScreenName: String
+    
     let logoImage: UIImage?
     let title: String
     let videoLinkLabel: String
-
-    required init(localizationServices: LocalizationServices) {
+    let youtubeVideoId: String
+    
+    required init(flowDelegate: FlowDelegate?, localizationServices: LocalizationServices, analyticsContainer: AnalyticsContainer, analyticsScreenName: String) {
```

Comment:
Hey @reldredge71 let's require the FlowDelegate on initialization just because it is required for this VIewModel to function properly by reporting those steps back to the Flow.  
Same with on ```OnboardingTutorialCustomViewBuilder```.